action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     description: "Name of the workflow to be ignored (the one who is waiting for the rest)"
     required: false
     default: ""
+  allowed-conclusions:
+    description: "Array of allowed conclusions"
+    required: false
+
```

Comment:
Please add the argument in the line 43:
`- ${{ inputs.allowed-conclusions }}`

Reply:
and please add a default value here as well. The defaults in the `config_accessor`s only apply when no values are given.

Reply:
Sorry! Didn't see that.